kubernetes/linux/Dockerfile.multiarch

```diff
@@ -11,12 +12,35 @@ COPY source /src/source
 RUN cd /src/build/linux && make arch=${TARGETARCH}
 
 
-FROM ${CI_BASE_IMAGE} AS base_image
+FROM ${MARINER_BASE_IMAGE} AS builder
 ARG TARGETOS TARGETARCH
 MAINTAINER OMSContainers@microsoft.com
 LABEL vendor=Microsoft\ Corp \
     com.microsoft.product="Azure Monitor for containers"
 ENV tmpdir /opt
+
+RUN tdnf install -y dnf && dnf -y clean expire-cache && dnf -y check-update
+RUN tdnf install -y build-essential wget openssl curl sudo net-tools cronie rsyslog vim dmidecode gnupg make logrotate busybox gawk tar && rm -rf /var/lib/apt/lists/*
+RUN mkdir /busybin && busybox --install /busybin
+
+COPY --from=golang-builder /src/kubernetes/linux/Linux_ULINUX_1.0_*_64_Release/docker-cimprov-*.*.*-*.*.sh $tmpdir/
+COPY kubernetes/linux/setup.sh kubernetes/linux/main.sh kubernetes/linux/defaultpromenvvariables kubernetes/linux/defaultpromenvvariables-rs kubernetes/linux/defaultpromenvvariables-sidecar kubernetes/linux/mdsd.xml kubernetes/linux/envmdsd kubernetes/linux/logrotate.conf $tmpdir/
+COPY kubernetes/linux/mariner-official-extras.repo /etc/yum.repos.d/
+
+# default value will be overwritten by pipeline 
+ARG IMAGE_TAG=3.1.4

```

Comment:
i see duplicate variables IMAGE_TAG & AGENT_VERSION.  can you please clean up the duplicate vars?